reveal/run.js

```diff
@@ -107,7 +107,16 @@ eslint no-param-reassign: [
         { src: 'reveal/plugins/zenika-footer/zenika-footer.js', condition() { return !!document.body.classList; } },
         { src: `${config.revealModule}plugin/math/math.js`, async: true },
       ],
-    });
+    };
+
+
+    fileExists('/framework/index.js')
+        .then((script) => {
+          /* eslint no-eval: "off", no-undef: "off" */
+          eval(script);
+          Reveal.initialize(overrideOptions(defaultOptions));
```

Comment:
This will crash if the file does not properly declare this function. A warning would be better. Using `window.overrideOptions` would also avoid the disabling of the no-undef eslint rule.

Reply:
There is a `catch` after so it's OK. I added a console.warn in the catch so that the error is visible in the console.